configure

```diff
@@ -104,13 +123,20 @@ if (config === 'production') {
     .description(`browserify ${excludes} $in -o $out`)
 }
 
-rule = ninja.rule('browserify-require')
-if (config === 'production') {
-  rule.run(`${browserify} ${requires} $in | ${uglifyjs} > $out`)
-} else {
-  rule.run(`${browserify} ${requires} $in -o $out`)
+function browserifyDependencies(dependencies) {
+  if (config === 'production') {
+    rule.run(`${browserify} ${dependencies} $in | ${uglifyjs} > $out`)
+  } else {
+    rule.run(`${browserify} ${dependencies} $in -o $out`)
+  }
 }
 
+rule = ninja.rule('browserify-require')
+browserifyDependencies(requires)
+
+rule = ninja.rule('browserify-submit')
+browserifyDependencies(submit)
+
```

Comment:
This is making me uneasy. I think you should pass the `rule` into the function and not rely on a `rule` global. Maybe even just create the `rule` within `browserifyDependencies`?

Reply:
Updated this to create the rule inside browserifyDependencies.